### DISCUSSION TITLE
Inject a Placeholder variable into Channel Form field

### DISCUSSION THREAD

**Discussion by ic360-com** [ExpressionEngine/ExpressionEngine#2921] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
I'd like to be able to easily inject a Placeholder variable into a Channel Form field, without constructing the form field manually.

**Describe the solution you'd like**
For example:
{field:order_shipping_tracking placeholder="Tracking No"}

I guess this would only work where form fields can accept Placeholders (text input, textarea).

**Describe alternatives you've considered**
I wonder whether this should become part of the Rules parameter, or a standalone feature?
rules:my_field_name=

**Comment by intoeetive** [comment]
Adding this as a parameter in channel form would require a lot of refactoring.

The proposed solution is to add Placeholder as setting for textarea and text fields; which would be effective both for CP and Channel Form: https://github.com/ExpressionEngine/ExpressionEngine/pull/4656
